src/serviceFacades/instantlaunches.js

```diff
@@ -358,14 +356,29 @@ export const instantlyLaunch = ({ instantLaunch, resource, output_dir }) => {
     // Contains the Promises that resolve to the data needed to perform a job submission.
     const promiseList = [
         savedLaunchPromise,
-        getAppInfo({ launchId: savedLaunchId }).catch((e) => console.log(e)),
+        getAppInfo({ launchId: savedLaunchId }),
```

Comment:
Not sure why the errors from these promises were getting caught and ignored, but I removed those catches so that errors from those endpoints can be properly displayed in the UI (rather than causing undefined `ql` or `app` exceptions below).